junit4/src/main/java/org/jboss/weld/junit4/WeldInitiator.java

```diff
@@ -213,6 +219,35 @@ private WeldInitiator(Weld weld, List<Object> instancesToInject, Set<Class<? ext
         super(weld, instancesToInject, scopesToActivate, beans, resources, ejbFactory, persistenceUnitFactory, persistenceContextFactory);
     }
 
+    /**
+     * Returns a {@link MethodRule} that can be used as a {@link Rule @Rule} to inject into the test class instance.
```

Comment:
```suggestion
     * Returns a {@link MethodRule} that can be used as a {@link Rule} to inject into the test class instance.
```

Reply:
Imho that decreases readability, but if you want it like that.
I guess then also for `{@code @Rule}` and `{@link ClassRule @ClassRule}`?